templates/named.conf.erb

```diff
@@ -8,11 +8,6 @@ logging {
                 severity dynamic;
         };
 };
-zone "." IN {
-       type hint;
-       file "<%= @root_hint %>";
```

Comment:
Hmm. I'm not sure what to do here.
On the one hand, we have this `@root_hint` parameter. On redhat, does /etc/named.rfc1912.zones have the same root zone hint?

Managing this named.conf file has opened up a can of worms. I think we have to go whole-hog here and manage everything, otherwise we are going to get deeper into a bunch of distro differences.

I agree that this is duplicated for the debian case.